apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -993,7 +993,10 @@ export class CaseService {
   }
 
   async addCompletedCaseToQueue(caseId: string): Promise<string> {
-    return this.queueService.add({ type: MessageType.CASE_COMPLETED, caseId })
+    return await this.queueService.add({
+      type: MessageType.CASE_COMPLETED,
+      caseId,
+    })
```

Comment:
Neccessary? Some recommend skipping await when you can return a "compatible" promise.

Reply:
Agree, there is no need to wait.